lib/fortnox/api/models/customer.rb

```diff
@@ -161,7 +161,7 @@ class Customer < Fortnox::API::Model::Base
         attribute :project, Types::Nullable::String
 
         # SalesAccount  Sales account of the customer, 4 digits
-        attribute :sales_account, Types::AccountNumber
+        attribute :sales_account, Types::SalesAccount
```

Comment:
Comment needs updating

Reply:
We should probably just get rid of these comments and refer to the documentation instead... These comments will just get old and I don't see the point in keeping them updated. Do you @d-Pixie?

Reply:
No, I agree. I almost wrote " ... or removed" yesterday 😄

Reply:
I'll change the comment for this PR so we leave this in a correct state.